src/features/peers/redux/actions.ts

```diff
@@ -0,0 +1,17 @@
+import { UPDATE_TOP_PEERS } from "./constants";
+import { ActionWithPayload } from "../../../store/actions";
+
+export type PeersData = {
+  symbol: string;
+  name: string;
```

Comment:
Q: does your peers data include names and symbols or just symbols?

Reply:
will put it into issues